components/gitpod-db/src/tables.ts

```diff
@@ -47,6 +47,12 @@ export class GitpodSessionTableDescriptionProvider implements TableDescriptionPr
 export class GitpodTableDescriptionProvider implements TableDescriptionProvider {
     readonly name = "gitpod";
     protected readonly tables: TableDescription[] = [
+        {
```

Comment:
Feels a bit risky to include this in the same PR as the migration. Are you happy with the risks involved in case the migration doesn't go through but db-sync gets rolled out?

Reply:
Good point - I'll leave the migration + config change in this PR for ease of review but I can split it into two before merging.

Reply:
I've removed the `db-sync` config change from this PR and I'll add it in a follow up.